wemake_python_styleguide/logic/collections.py

```diff
@@ -33,30 +46,33 @@ def normalize_dict_elements(node: ast.Dict) -> Sequence[ast.AST]:
 
 
 def sequence_of_node(
-    node_type: Tuple[Type[ast.stmt]],
+    node_types: Tuple[Type[_NodeType]],
```

Comment:
It should be `Tuple[Type[_NodeType], ...]`

Reply:
done